### ISSUE TITLE
install_stock script failing due to apt-key.gpg curl error

### ISSUE THREAD

**Issue by JooyoungPark73** [vhive-serverless/vHive#682]
https://github.com/vhive-serverless/vHive/blob/cc70f39715d76c49ede5e9325e5c7b364478f11e/scripts/install_stock.sh#L49

The referenced script fails to execute because of a faulty instruction
I am currently using cloudlab's mass-rs440 instance.

The error message is
```
gpg: no valid OpenPGP data found.
```

In case of the `curl` instruction at the former part
```
nehalem@node-0:/tmp$ curl --silent --show-error https://packages.cloud.google.com/apt/doc/apt-key.gpg

<!DOCTYPE html>
<html lang=en>
  <meta charset=utf-8>
  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
  <title>Error 500 (Internal Server Error)!!1</title>
  <style>
    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/logos/errorpage/error_logo-150x54.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/logos/errorpage/error_logo-150x54-2x.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/logos/errorpage/error_logo-150x54-2x.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/logos/errorpage/error_logo-150x54-2x.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}
  </style>
  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>
  <p><b>500.</b> <ins>That’s an error.</ins>
  <p>  <ins>That’s all we know.</ins>
```

The error is not always generated, so you might have to repeat them for a few times.
And it seems to be google's problem.
@ustiugov @cvetkovic are there any solutions?

**Comment by Lightxyz** [comment]
You can try using the modified installation script on this branch:
https://github.com/Lightxyz/vHive/blob/fix_install_scripts/scripts/install_stock.sh
It removes the curl command and adds [trusted=yes] so that GPG keys are not checked when installing from that repository, so simply adding line 49 from that script to the script you are using should work, until Google fixes the issue.

**Comment by ustiugov** [comment]
@Lightxyz can you upstream this workaround? we are not a production system, we don't have to check the keys

**Comment by Lightxyz** [comment]
@ustiugov Submitted a pull request :)

**Comment by leokondrashov** [comment]
@JooyoungPark73 Was this fixed by #691 and #762?

**Comment by JooyoungPark73** [comment]
@leokondrashov  yes it is fixed by #762
